aztk/spark/models/plugins/jupyter_lab/jupyter_lab.sh

```diff
@@ -42,7 +42,7 @@ if  [ "$AZTK_IS_MASTER" = "true" ]; then
     "env": {
         "SPARK_HOME": "$SPARK_HOME",
         "PYSPARK_PYTHON": "python",
-        "PYSPARK_SUBMIT_ARGS": "--master spark://$MASTER_IP:7077 pyspark-shell"
+        "PYSPARK_SUBMIT_ARGS": "--master spark://$AZTK_MASTER_IP:7077 pyspark-shell"
```

Comment:
This seems to actually be a bug, though. `$AZTK_MASTER_IP` is the correct environment variable.